sdk/eventhub/azure-eventhub/README.md

```diff
@@ -62,6 +52,10 @@ consumer_client = EventHubConsumerClient.from_connection_string(connection_str,
 
 ```
 
+- For the Event Hubs client library to interact with an Event Hub, it will need to understand how to connect and authorize with it.
```

Comment:
This statement is not relevant to connection string. So I think we can remove it.